lib/extensions/ar_migration.rb

```diff
@@ -10,19 +21,21 @@ def self.discover_schema_migrations_ran_class
   end
 
   def self.update_local_migrations_ran(version, direction)
-    return unless schema_migrations_ran_class = discover_schema_migrations_ran_class
-
-    new_migrations = ActiveRecord::SchemaMigration.normalized_versions
-    new_migrations << version if direction == :up
+    return unless schema_migrations_ran_exists?
 
-    to_add = (new_migrations - schema_migrations_ran_class.pluck(:version))
-    puts "Seeding :schema_migrations_ran table..." if to_add.size > 1
+    if direction == :up
+      if version == SCHEMA_MIGRATIONS_RAN_MIGRATION
+        to_add = ActiveRecord::SchemaMigration.normalized_versions << version
+      else
+        to_add = [version]
+      end
 
-    to_add.each do |v|
-      schema_migrations_ran_class.find_or_create_by(:version => v)
+      to_add.each do |v|
+        ActiveRecord::Base.connection.execute("INSERT INTO schema_migrations_ran (version, created_at) VALUES ('#{v}', '#{Time.now.utc.iso8601(6)}')")
+      end
+    else
+      ActiveRecord::Base.connection.execute("DELETE FROM schema_migrations_ran WHERE version = '#{version}'")
```

Comment:
Not for this PR, but I think this might all be cleaner if the queries are extracted to a helper module...I'm picturing something like

```ruby
module SchemaMigrationRan
  def self.create!(version)
    ActiveRecord::Base.connection.execute("INSERT INTO schema_migrations_ran (version, created_at) VALUES ('#{version}', '#{Time.now.utc.iso8601(6)}')")
  end

  def self.delete!(version)
    ActiveRecord::Base.connection.execute("DELETE FROM schema_migrations_ran WHERE version = '#{version}'")
  end

  # Eventually later the other query commands
end

# Then here it's just:

    if direction == :up
      if version == SCHEMA_MIGRATIONS_RAN_MIGRATION
        ActiveRecord::SchemaMigration.normalized_versions.each { |v| SchemaMigrationRan.create!(v) } 
      end
      SchemaMigrationRan.create!(version)
    else
      SchemaMigrationRan.delete!(version)
    end